textproc/libunistring/Portfile

```diff
@@ -12,8 +12,8 @@ license             GPL-3
 maintainers         nomaintainer
 description         library for manipulating Unicode strings
 long_description    Text files are nowadays usually encoded in Unicode, and may consist of \
-                    very different scripts – from Latin letters to Chinese Hanzi –, with many \
-                    kinds of special characters – accents, right-to-left writing marks, \
+                    very different scripts - from Latin letters to Chinese Hanzi - with many \
+                    kinds of special characters - accents, right-to-left writing marks, \
                     hyphens, Roman numbers, and much more. But the POSIX platform APIs for \
```

Comment:
Any idea what happened here? Did you accidentally replace ASCII dashes with Unicode? Regardless, I'd revert this particular part of the change as it's not there to fix Tiger.